<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Timeline</class>
 <widget class="QDialog" name="Timeline">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timeline</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="seismicityTitleLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Seismicity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="SeismicityPlotWidget" name="seismic_data_plot" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="hydraulicsTitleLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Hydraulics</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="dataComboBox">
         <item>
          <property name="text">
           <string>Top hole pressure</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="HydraulicsPlotWidget" name="hydraulic_data_plot" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <zorder>hydraulic_data_frame</zorder>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="forecastsTitleLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Forecasts</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="measureComboBox">
         <item>
          <property name="text">
           <string>IS Rate</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="isModelComboBox">
         <item>
          <property name="text">
           <string>ETAS</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="RateForecastPlotWidget" name="forecasts_data_plot" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SeismicityPlotWidget</class>
   <extends>QWidget</extends>
   <header>ui.views.plots</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HydraulicsPlotWidget</class>
   <extends>QWidget</extends>
   <header>ui.views.plots</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RateForecastPlotWidget</class>
   <extends>QWidget</extends>
   <header>ui.views.plots</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
